sig-docs/charter.md

```diff
@@ -0,0 +1,76 @@
+# SIG Docs Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README] and adheres to the Roles and Organization Management specified in [sig-governance].
+
+## Scope
+
+SIG Docs is responsible for Kubernetes documentation.
+
+The responsibility for feature documentation lies with feature developers. SIG Docs sets standards for feature documentation, provides clear paths for docs contribution, and coordinates documentation updates during quarterly releases.
+
+We maintain the Kubernetes website's infrastructure and tooling.
+
+We create subprojects as needed to handle specific aspects of Kubernetes documentation.
+
+### In scope
+
+SIG [readme]
+
+#### Code, Binaries and Services
+
+- The kubernetes.io website:
+
+  - Infrastructure and tooling
+  - Internationalization and localization support
+  - Documentation and release note training for other sigs
+  - Generated reference documentation and review of related strings in code for:
+    - The core Kubernetes APIs
+    - Kubernetes command-line tools, including but not limited to kubectl
+    - Kubernetes setup tools
+    - The Federation API
+
+See the next section for limitations on how the SIG helps with related efforts on other projects.
+
+#### Cross-cutting and Externally Facing Processes
+
+- [Standards for content][standards]: style guide, contributor guide, PR reviews
+  - Note that the SIG is not responsible for branding any content related to the Kubernetes organization. We are sometimes informally invited to consult on branding and design decisions, but we have no formal responsibility for such work.
+- [Coordinating docs contributions for quarterly releases][docs-release]
+- Blog PR management
+- Reorganization and rewrite of technical content to improve UX and technical accuracy based on current and recent releases, based on:
```

Comment:
> Reorganization and rewrite of technical content to improve UX and technical accuracy based on current and recent releases, based on:

Is this just: `Make direct docs contributions to improve quality and address issues related to: UX and technical accuracy.`

Reply:
@pwittrock Yes. I've revised this section to make that clearer.